src/gax/src/paginator.rs

```diff
@@ -84,13 +86,26 @@ where
         ItemPaginator::new(self)
     }
 
+    /// Returns the next mutation of the wrapped stream.
+    ///
+    /// Enable the `unstable-stream` feature to interact with a [`futures::stream::Stream`].
+    ///
+    /// [`futures::stream::Stream`]: https://docs.rs/futures/latest/futures/stream/trait.Stream.html
+    pub async fn next_stable(&mut self) -> Option<Result<T, E>> {
```

Comment:
Can we arrange things so this function is called `next()`?  If not, can we call it `next_page()`?

Reply:
Calling this function `next()` seems to work.